tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/KafkaConsumerConstructorIntegration.cs

```diff
@@ -30,16 +30,31 @@ public class KafkaConsumerConstructorIntegration
     {
         if (Tracer.Instance.Settings.IsIntegrationEnabled(KafkaConstants.IntegrationId))
         {
+            string groupId = null;
+            string bootstrapServers = null;
+
             foreach (var kvp in consumer.Config)
             {
                 if (string.Equals(kvp.Key, "group.id", StringComparison.Ordinal))
                 {
                     if (!string.IsNullOrEmpty(kvp.Value))
                     {
-                        // Save the map between this consumer and a consumer group
-                        return new CallTargetState(scope: null, state: kvp.Value);
+                        groupId = kvp.Value;
                     }
                 }
+                else if (string.Equals(kvp.Key, "bootstrap.servers", StringComparison.Ordinal))
+                {
+                    if (!string.IsNullOrEmpty(kvp.Value))
+                    {
+                        bootstrapServers = kvp.Value;
+                    }
+                }
+            }
+
+            if (groupId is not null)
```

Comment:
I take it it's ok if `bootstrapServers` is `null`? I assume this is added to keep the behaviour identical, but I wonder if that's necessary, and that it should actually be: 

```suggestion
            if (groupId is not null && bootstrapServers is not null)
```

Reply:
So it turns out that in the ConsumerConfig `groupId` is a hard-requirement, but `bootstrapServers` is not, so I'd like to keep this as-is. If the current conditional is okay with you, can I get a 👍🏼 on this comment?